src/views/Inventory.vue

```diff
@@ -42,58 +47,80 @@
 
 <script lang="ts">
 import Vue from "vue";
+import { mapActions, mapGetters } from "vuex";
 import Component from "vue-class-component";
 import ViewTitle from "@/components/ViewTitle.vue";
 import InventoryActions from "@/components/InventoryActions.vue";
 import EditCard from "@/components/EditCard.vue";
 import { Status, Furniture } from "@/data/Furniture";
-import * as firebase from "firebase/app";
-import "firebase/firestore";
+
+const namespace = "inventory";
 
 @Component({
   components: {
     ViewTitle,
     InventoryActions,
     EditCard,
   },
+  computed: mapGetters(namespace, {
+    inventory: "getInventory",
+    current: "getCurrent",
+  }),
+  methods: mapActions(namespace, [
+    "bindInventory",
+    "setCurrent",
+    "clearCurrent",
+  ]),
 })
 export default class Inventory extends Vue {
-  status = Status;
+  current!: Furniture;
+
```

Comment:
Would you need to also define inventory down here as well?

Reply:
Ah, good question! If I wanted to use the `inventory` variable within the class code below, I would have to declare it as a property within the class. But, I only use it in the `<template>` above, so I don't need it.

Basically, repeating the declaration inside the class is only needed in order for functions/stuff inside the class to use it.

Reply:
Gotcha. And for my own understanding, you used inventory in the v-data-table part?

Reply:
yup